forge-gui/res/cardsfolder/upcoming/wheel_of_potential.txt

```diff
@@ -0,0 +1,14 @@
+Name:Wheel of Potential
+ManaCost:2 R
+Types:Sorcery
+A:SP$ PutCounter | Defined$ You | CounterType$ ENERGY | CounterNum$ 3 | SubAbility$ ChooseX | StackDescription$ REP You get_{p:You} gets & you_ | SpellDescription$ You get {E}{E}{E} (three energy counters), then you may pay X {E}.,,,,,,
+SVar:ChooseX:DB$ ChooseNumber | Max$ Count$YourCountersEnergy | ListTitle$ Choose amount for X | SubAbility$ Pay | StackDescription$ None
+SVar:Pay:DB$ Pump | UnlessCost$ Mandatory PayEnergy<X> | UnlessPayer$ You | UnlessSwitched$ True | SubAbility$ Repeat | StackDescription$ None
+SVar:Repeat:DB$ RepeatEach | RepeatPlayers$ Player | RepeatSubAbility$ Draw | SubAbility$ Effect | SpellDescription$ Each player may exile their hand and draw X cards.
+SVar:Draw:DB$ Draw | UnlessCost$ ExileFromHand<1/All> | UnlessPayer$ Remembered | UnlessSwitched$ True | Defined$ Remembered | NumCards$ X
+SVar:Effect:DB$ Effect | ConditionCheckSVar$ X | ConditionSVarCompare$ GE7 | RememberObjects$ ExiledWith.YouOwn | StaticAbilities$ Play | Duration$ UntilTheEndOfYourNextTurn | ForgetOnMoved$ Exile | SubAbility$ DBCleanup | SpellDescription$ If X is 7 or more, you may play cards you own exiled this way until the end of your next turn.
+SVar:Play:Mode$ Continuous | MayPlay$ True | EffectZone$ Command | Affected$ Card.YouOwn+IsRemembered | AffectedZone$ Exile | Description$ You may play cards you own exiled this way until the end of your next turn.
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
```

Comment:
nothing to cleanup it seems?

Though I'm not sure yet about your approach of (ab)using what's meant for linked abilities to avoid it (for one you have to consider a lot of corner cases that would allow other exiling to happen)
I'll give myself time until the rest is sorted out to mull it over

Reply:
I think making it not a cost might help me sort this